site/layouts/blog/list.html

```diff
@@ -13,10 +13,18 @@
     <div class="container">
         <div class="row">
             <div class="col-md-9 doc-content">
-            <h3>Mattermost Developer Blog</h3>
-            {{ range first 10 .Data.Pages }}
-                {{ .Render "summary" }}
-            {{ end }}
+                <h3>Mattermost Developer Blog</h3>
+                {{ range (.Paginate .Data.Pages).Pages }}
+                    {{ .Render "summary" }}
+                {{ end }}
+                <div class="blog-pagination">
+                    {{ if .Paginator.HasNext }}
+                        <a href="{{ .Paginator.Next.URL }}"> &laquo; Previous Posts</a>
```

Comment:
Maybe using the standard bootstrap buttons would help reduce the number of different UI elements.  Something like:
```suggestion
                        <a class="btn btn-primary btn-sm" href="{{ .Paginator.Next.URL }}"> &laquo; Previous Posts</a>
```

Reply:
Thanks. I tried suggested style, and you can see it on https://kaakaa.github.io/mattermost-developer-documentation/blog/page/2/.